dtfs-central-api/src/services/portal/deal.service.add-deal-cancelled-activity.test.ts

```diff
@@ -38,43 +42,105 @@ describe('PortalDealService - addGefDealCancelledActivity', () => {
   });
 
   describe(`when dealType is ${DEAL_TYPE.GEF}`, () => {
-    it('should call addPortalActivity', async () => {
-      // Arrange
-      addPortalActivityMock.mockResolvedValue({ matchedCount: 1 });
-
+    beforeEach(() => {
       jest.spyOn(PortalActivityRepo, 'addPortalActivity').mockImplementation(addPortalActivityMock);
+    });
 
-      const mockDeal = deal;
-      mockDeal.dealSnapshot.dealType = DEAL_TYPE.GEF;
+    const mockDeal = deal;
+    mockDeal.dealSnapshot.dealType = DEAL_TYPE.GEF;
```

Comment:
This is modifying `deal` since `const mockDeal = deal;` copies by reference.
As I commented elsewhere, we probably want to re-instantiate this in every test to avoid the mock deal object beign modified

Reply:
This is now outdated, deal object is no longer required.